tsdb/index/tsi1/log_file.go

```diff
@@ -1300,6 +1294,20 @@ func (m *logMeasurement) seriesIDSet() *tsdb.SeriesIDSet {
 	return ss
 }
 
+func (m *logMeasurement) hasSeries(ss *tsdb.SeriesIDSet) bool {
+	if m.seriesSet != nil {
```

Comment:
@StoneYunZhao can you just add a comment here indicating one of these is an optimised fast path?